example_code/applications/vending_machine/.cargo/config.toml

```diff
@@ -0,0 +1,16 @@
+[target.wasm32-unknown-unknown]
```

Comment:
I think this dot file is unnecessary and generated by `cargo` but not intended to be checked in to source

Reply:
Ok, will have it removed and add it to the .gitignore